core/src/main/java/com/orange/ouds/core/theme/OudsTheme.kt

```diff
@@ -21,10 +21,12 @@ import androidx.compose.runtime.staticCompositionLocalOf
 import com.orange.ouds.theme.OudsColors
 import com.orange.ouds.theme.OudsCustomTheme
 import com.orange.ouds.theme.tokens.components.OudsComponentsTokens
+import com.orange.ouds.theme.tokens.semantic.OudsOpacityTokens
 
 private fun customThemeError(message: Any): Nothing = error("OudsTheme not found. $message")
 
 internal val LocalColors = staticCompositionLocalOf<OudsColors> { customThemeError("LocalColors CompositionLocal not present") }
+private val LocalOpacityTokens = staticCompositionLocalOf { OudsOpacityTokens() }
```

Comment:
Same comment as #52:

I am not sure we should return `OudsOpacityTokens()` by default here. If we change the default value in `OudsCustomTheme` the two values won't be aligned anymore.

Reply:
What do you suggest?